### PR TITLE
Update PPP

### PR DESCRIPTION
**Pull request by jamz903**

### CODE REVIEW

File: docs/team/jamz903.md [hunk 1]
```diff
@@ -6,32 +6,56 @@ title: Jamie's Project Portfolio Page
...
-* **Code contributed**: [to be added]()
 
 * **Project management**:
-    * Managed releases `v1.1` (1 release) on GitHub
+    * Managed releases `v1.2a` (1 release) on GitHub
+
 
 * **Enhancements to existing features**:
-    * *to be added*
+    * Enhance Style & Display of Ui to match prototype [[PR #76]](https://github.com/AY2324S1-CS2103-T16-3/tp/pull/76)
```
Comment (woojiahao) [inline docs/team/jamz903.md:41]:
```suggestion
    * Enhance Style & Display of UI to match prototype [[PR #76]](https://github.com/AY2324S1-CS2103-T16-3/tp/pull/76)
```

Comment (woojiahao) [reply docs/team/jamz903.md:41]: Super minor nit haha

**Review by woojiahao — CHANGES_REQUESTED** [review]
Awesome @jamz903 Do you want to update your about me role/responsibilities too? Also minor spelling nit

**Review by woojiahao — APPROVED** [review]
LGTM

**Review by sp4ce-cowboy — APPROVED** [review]
LGTM, appreciate the shout out! Btw I think you can add refactor UI as well (in the new features part), since you did the initial refactoring from AB3 to UniCa$h (and the UI Mockup), in my PPP I'll credit you as well for the UI refactoring.

**Review by sp4ce-cowboy — APPROVED** [review]
LGTM!

**Pull request merged**